parts/openshift/unstable/openshiftmasterscript.sh

```diff
@@ -95,14 +95,20 @@ for i in /etc/origin/master/master-config.yaml /tmp/bootstrapconfigs/* /tmp/ansi
     sed -i "s|HOSTNAME|${HOSTNAME}|g;" $i
 done
 
+mkdir -p /root/.kube
+
+for loc in /root/.kube/config /etc/origin/node/bootstrap.kubeconfig /etc/origin/node/node.kubeconfig; do
+  cp /etc/origin/master/admin.kubeconfig "$loc"
+done
+
 # note: ${SERVICE_TYPE}-node crash loops until master is up
```

Comment:
> `# note: ${SERVICE_TYPE}-node crash loops until master is up`

is this comment still true? if not, remove

Reply:
removed.